src/templates/finops-hub/modules/hub.bicep

```diff
@@ -163,8 +163,10 @@ var dataFactoryName = replace(
 
 // Do not reference the dataExplorer deployment directly or indirectly to avoid a DeploymentNotFound error
 var deployDataExplorer = !empty(dataExplorerName)
-var dataExplorerUri = !deployDataExplorer ? '' : dataExplorer.outputs.clusterUri
-var dataExplorerIngestionDb = !deployDataExplorer ? '' : dataExplorer.outputs.ingestionDbName
+var safeDataExplorerName = !deployDataExplorer ? '' : dataExplorer.outputs.clusterName
+var safeDataExplorerUri = !deployDataExplorer ? '' : dataExplorer.outputs.clusterUri
+var safeDataExplorerId = !deployDataExplorer ? '' : dataExplorer.outputs.clusterId
+var safeDataExplorerIngestionDb = !deployDataExplorer ? '' : dataExplorer.outputs.ingestionDbName
```

Comment:
nit: What makes any of these "safe"? I fear that word implies something that isn't true in this case. Is there's anything "safe" about these values given they're the raw values?

Reply:
We need something in the string value and dataExplorer is optional.